packages/shared/package.json

```diff
@@ -121,7 +121,7 @@
     "lodash.clonedeep": "^4.5.0",
     "node-fetch": "^2.6.6",
     "react-markdown": "^8.0.7",
-    "react-onesignal": "^2.0.4",
+    "react-onesignal": "^3.0.1",
```

Comment:
Would we also need to update the API side?

Reply:
Good question. This should be independent in terms of usage, but since there is nothing wrong yet with the API it should be good. Probably worth to check as well if we need to upgrade it as well.